actors/verifreg/tests/harness/mod.rs

```diff
@@ -205,12 +205,9 @@ impl Harness {
         acc.assert_empty();
     }
 
-    // TODO this should be implemented through a call to verifreg but for now it modifies state directly
```

Comment:
I think this TODO still holds up.  This way is convenient but we  for coverage we should ideally exercise calling the receiver hook using `receive_tokens` harness method to create allocations in test.

Reply:
I've restored the comment. Generally I'd agree, but it's close here. And for `create_claim` that I added I think even more reasonable to mutate the state rather than go through the public procedure for creating a claim.